liferay-echo/liferay-portal
# LPS-196851 Add Copy with permissions for Display Pages

## Motivation

Add options for copy with and without permissions in the dropdown for Display Pages

[Link to ticket](https://liferay.atlassian.net/browse/LPS-196851)

## Test scenario 1 
1. Create a new Display Page.
2. Change the permissions for this Display Page.
3. Make a Copy with permissions and a copy without them. Check that they are copied or not depending on the case.

## Results

https://github.com/liferay-echo/liferay-portal/assets/93203423/c0e96ef8-b14b-44cd-9990-d0065ab9c490
